tools/mason/MasonUtils.chpl

```diff
@@ -132,11 +142,12 @@ proc getSpackResult(cmd, quiet=false) : string throws {
   try {
 
 
-    var prefix = "export SPACK_ROOT=" + MASON_HOME + "/spack" +
-    " && export PATH=$SPACK_ROOT/bin:$PATH" +
-    " && source $SPACK_ROOT/share/spack/setup-env.sh && ";
+    var prefix = "export SPACK_ROOT=" + SPACK_ROOT +
+    " && export PATH=\"$SPACK_ROOT/bin:$PATH\"" +
```

Comment:
Actually, why do we need to set the PATH? I believe Spack's setup-env.sh handles that.